lib/status/status.go

```diff
@@ -48,12 +48,18 @@ import (
 // FromCluster collects cluster status information.
 // The function returns the partial status if not all details can be collected
 func FromCluster(ctx context.Context, operator ops.Operator, cluster ops.Site, operationID string) (status *Status, err error) {
+	site, err := operator.GetLocalSite()
```

Comment:
You already have the `cluster` object passed in this function, so I don't think you need to retrieve it again?